# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: testing/web-platform/meta/scroll-to-text-fragment/find-range-from-text-directive.html.ini
# Commit: 85883a720dab
# Full Hash: 85883a720dab61e82bcdb7391236dc06427466ee
# Author: moz-wptsync-bot <wptsync@mozilla.com>
# Date: 2021-04-07 15:47:02
# Regressor Bug: 1702830
# File Overlap Count: 1
# Description:
#   Bug 1702830 - [wpt-sync] Update web-platform-tests to 336ec39f06d15863a4af6b2aa6592530b37a5aee, a=testonly
#   
#   MANUAL PUSH: wpt sync bot
#   
#   wpt-head: 336ec39f06d15863a4af6b2aa6592530b37a5aee
# ==============================================================================

diff -r acef12d74007 -r 85883a720dab testing/web-platform/meta/scroll-to-text-fragment/find-range-from-text-directive.html.ini
--- a/testing/web-platform/meta/scroll-to-text-fragment/find-range-from-text-directive.html.ini	Fri Apr 02 22:03:46 2021 +0000
+++ b/testing/web-platform/meta/scroll-to-text-fragment/find-range-from-text-directive.html.ini	Fri Apr 02 22:06:52 2021 +0000
@@ -1,14 +1,16 @@
 [find-range-from-text-directive.html]
-  expected: TIMEOUT
+  expected:
+    if os == "mac": OK
+    TIMEOUT
   [inverted range search.]
     expected:
-      if os == "win": ["PASS", "NOTRUN", "TIMEOUT"]
       if os == "android": ["NOTRUN", "PASS", "TIMEOUT"]
       [PASS, TIMEOUT, NOTRUN]
 
   [Suffix comes before |end|.]
     expected:
-      if os == "linux": ["TIMEOUT", "NOTRUN", "PASS"]
+      if os == "mac": ["PASS", "TIMEOUT", "NOTRUN"]
+      if os == "linux": ["PASS", "NOTRUN", "TIMEOUT"]
       [NOTRUN, PASS, TIMEOUT]
 
   [Multiple overlapping prefixes.]
@@ -21,7 +23,8 @@
 
   [Suffix must be end bounded.]
     expected:
-      if os == "linux": ["NOTRUN", "PASS", "TIMEOUT"]
+      if os == "linux": ["PASS", "NOTRUN", "TIMEOUT"]
+      if os == "mac": PASS
       NOTRUN
 
   [non-existent exact match.]
@@ -32,7 +35,6 @@
   [non-existent range match.]
     expected:
       if os == "android": ["NOTRUN", "PASS", "TIMEOUT"]
-      if os == "win": ["PASS", "NOTRUN", "TIMEOUT"]
       [PASS, TIMEOUT, NOTRUN]
 
   [Multiple overlapping one letter prefixes.]
@@ -40,50 +42,53 @@
 
   [Content appears between match and suffix.]
     expected:
+      if os == "mac": PASS
       if os == "linux": ["NOTRUN", "TIMEOUT"]
       NOTRUN
 
   [overlapping exact matches with suffix.]
     expected:
-      if os == "win": ["FAIL", "NOTRUN", "TIMEOUT"]
       if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
       [FAIL, TIMEOUT, NOTRUN]
 
   [Match with no suffix.]
     expected:
-      if os == "linux": ["FAIL", "TIMEOUT", "NOTRUN"]
-      [NOTRUN, FAIL, TIMEOUT]
+      if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
+      if os == "win": ["NOTRUN", "FAIL", "TIMEOUT"]
+      if os == "mac": ["FAIL", "TIMEOUT", "NOTRUN"]
+      [FAIL, NOTRUN, TIMEOUT]
 
   [Match text after prefix.]
     expected: FAIL
 
   [Search invisible content between |end| and suffix.]
-    expected: [NOTRUN, FAIL, TIMEOUT]
+    expected:
+      if os == "mac": ["FAIL", "TIMEOUT", "NOTRUN"]
+      if os == "linux": ["FAIL", "NOTRUN", "TIMEOUT"]
+      [NOTRUN, FAIL, TIMEOUT]
 
   [Range with preceeding suffix.]
     expected:
-      if os == "mac": ["TIMEOUT", "NOTRUN", "PASS"]
-      if os == "linux": ["PASS", "TIMEOUT", "NOTRUN"]
+      if os == "linux": ["PASS", "NOTRUN", "TIMEOUT"]
+      if os == "mac": ["PASS", "TIMEOUT", "NOTRUN"]
       [NOTRUN, PASS, TIMEOUT]
 
   [no suffix forces |end| to be end bounded.]
     expected:
       if os == "android": ["NOTRUN", "PASS", "TIMEOUT"]
-      if os == "win": ["PASS", "NOTRUN", "TIMEOUT"]
       [PASS, TIMEOUT, NOTRUN]
 
   [|end| must be start bounded even if full range is word bounded.]
     expected:
-      if os == "win": ["TIMEOUT", "NOTRUN", "PASS"]
-      if os == "linux": ["PASS", "TIMEOUT", "NOTRUN"]
+      if os == "win": ["TIMEOUT", "PASS", "NOTRUN"]
       if os == "android": ["NOTRUN", "PASS", "TIMEOUT"]
-      [PASS, NOTRUN, TIMEOUT]
+      [PASS, TIMEOUT, NOTRUN]
 
   [non-existent |end|.]
     expected:
-      if os == "linux": ["PASS", "TIMEOUT", "NOTRUN"]
-      if os == "mac": ["PASS", "NOTRUN", "TIMEOUT"]
-      [NOTRUN, PASS, TIMEOUT]
+      if os == "win": ["NOTRUN", "PASS", "TIMEOUT"]
+      if os == "android": ["NOTRUN", "PASS", "TIMEOUT"]
+      [PASS, TIMEOUT, NOTRUN]
 
   [suffix means |start| need not end on word boundary.]
     expected: FAIL
@@ -93,7 +98,6 @@
 
   [overlapping one letter exact matches with suffix.]
     expected:
-      if os == "win": ["FAIL", "NOTRUN", "TIMEOUT"]
       if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
       [FAIL, TIMEOUT, NOTRUN]
 
@@ -103,7 +107,6 @@
   [matching range search.]
     expected:
       if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
-      if os == "win": ["FAIL", "NOTRUN", "TIMEOUT"]
       [FAIL, TIMEOUT, NOTRUN]
 
   [Prefix need not end on a word boundary.]
@@ -125,23 +128,23 @@
   [Non-existent suffix.]
     expected:
       if os == "linux": ["NOTRUN", "PASS", "TIMEOUT"]
+      if os == "mac": PASS
       NOTRUN
 
   [Suffix need not be start bounded.]
     expected:
-      if os == "linux": ["NOTRUN", "FAIL", "TIMEOUT"]
+      if os == "mac": FAIL
+      if os == "linux": ["TIMEOUT", "NOTRUN", "FAIL"]
       NOTRUN
 
   [|end| must be start bounded.]
     expected:
       if os == "android": ["NOTRUN", "PASS", "TIMEOUT"]
-      if os == "win": ["PASS", "NOTRUN", "TIMEOUT"]
       [PASS, TIMEOUT, NOTRUN]
 
   [suffix means |end| need not be end bounded.]
     expected:
       if os == "android": ["NOTRUN", "FAIL", "TIMEOUT"]
-      if os == "win": ["FAIL", "NOTRUN", "TIMEOUT"]
       [FAIL, TIMEOUT, NOTRUN]
 
   [|start| must start on a word boundary.]